tests/js/client/shell/shell-check-metrics-move-noncluster.js

```diff
@@ -0,0 +1,112 @@
+      let http1ReqSum = getMetric("arangodb_request_body_size_http1_sum");
+      let http2ReqCount = getMetric("arangodb_request_body_size_http2_count");
+      let http2ReqSum = getMetric("arangodb_request_body_size_http2_sum");
+      let vstReqCount = getMetric("arangodb_request_body_size_vst_count");
+      let vstReqSum = getMetric("arangodb_request_body_size_vst_sum");
+      // Do a few requests:
+      for (let i = 0; i < 10; ++i) {
+        let res = arango.GET_RAW("/_api/version");
+        assertEqual(200, res.code);
+      }
+      // Note that GET requests do not have bodies, so they do not move
+      // the sum of the histogram. So let's do a PUT just for good measure:
+      let logging = arango.GET("/_admin/log/level");
+      let res = arango.PUT_RAW("/_admin/log/level", logging);
+      assertEqual(200, res.code);
+      // And get the values again:
+      let http1ReqCount2 = getMetric("arangodb_request_body_size_http1_count");
+      let http1ReqSum2 = getMetric("arangodb_request_body_size_http1_sum");
+      let http2ReqCount2 = getMetric("arangodb_request_body_size_http2_count");
+      let http2ReqSum2 = getMetric("arangodb_request_body_size_http2_sum");
+      let vstReqCount2 = getMetric("arangodb_request_body_size_vst_count");
+      let vstReqSum2 = getMetric("arangodb_request_body_size_vst_sum");
+      if (arango.protocol() === "vst") {
+        assertNotEqual(vstReqCount, vstReqCount2);
+        assertNotEqual(vstReqSum, vstReqSum2);
+      } else {
+        assertEqual(vstReqCount, vstReqCount2);
+        assertEqual(vstReqSum, vstReqSum2);
+      }
+      if (arango.protocol() === "http") {
+        assertNotEqual(http1ReqCount, http1ReqCount2);
+        assertNotEqual(http1ReqSum, http1ReqSum2);
+      } else {
+        assertEqual(http1ReqCount, http1ReqCount2);
+        assertEqual(http1ReqSum, http1ReqSum2);
+      }
+      if (arango.protocol() === "http2") {
+        assertNotEqual(http2ReqCount, http2ReqCount2);
+        assertNotEqual(http2ReqSum, http2ReqSum2);
+      } else {
+        assertEqual(http2ReqCount, http2ReqCount2);
+        assertEqual(http2ReqSum, http2ReqSum2);
+      }
+    },
+
+    testConnectionCountByProtocol: function () {
+      let http2ConnCount = getMetric("arangodb_http2_connections_total");
+      let vstConnCount = getMetric("arangodb_vst_connections_total");
+      let allConnCount = getMetric("arangodb_client_connection_statistics_connection_time_count");
+      // I have not found a reliable way to trigger a new connection from
+      // `arangosh`. `arango.reconnect` does not work since it is caching
+      // connections and the request timeout is not honoured for HTTP/2
+      // and VST by fuerte. The idle timeout runs into an assertion failure.
+      // Therefore, we must be content here to check that the connection
+      // count is non-zero for the currently underlying protocol:
+      //assertNotEqual(0, allConnCount);
```

Comment:
`allConnCount` is not used anymore by the test. Shall we remove it?

Reply:
I will fix this. My understanding was that this metric counts all connections, but this does not seem to be true. I will investigate and either delete or fix.

Reply:
Done.